Release Notes/Release Notes v2.9.0.md

```diff
@@ -1,3 +1,30 @@
 # .NET Driver Version 2.9.0 Release Notes
 
-This file is a placeholder.
\ No newline at end of file
+The main new features in 2.9.0 are:
+
+* Sharded transactions using the same API as replica set transactions
+* Sharded transactions pinning to a single mongos router
+* Convenient API for transactions
```

Comment:
This could use some elaboration.  Just calling it a convenient API (despite the spec name) is not informative.  The Java driver release notes, for example, say:

> The sessions API supports the ClientSession.withTransaction() method to conveniently run a transaction with automatic retries and at-most-once semantics.

Reply:
Adapted this with some tweaks for the C# driver